attest/activation.go

```diff
@@ -88,32 +98,68 @@ func (p *ActivationParameters) checkTPM12AKParameters() error {
 	return nil
 }
 
+// VerifyOpts specifies options passed to (*AttestationParameters).Verify()
+type VerifyOpts struct {
+	// SelfAttested set to true ensures that the attestation is self-signed,
+	// set to false ensures that the attestation is not self-signed.
+	SelfAttested bool
```

Comment:
This should be "InsecureSelfAttested" similar to "InsecureSkipVerify" https://golang.org/pkg/crypto/tls/#Config

Is there a reason we wanted to support this?

Reply:
AK's AttestationParameters are self-signed, so I keep it for this case. I'm fine with marking it insecure.

Reply:
I guess my point is this shouldn't be exposed to the user. We don't require the user to call Verify() for AKs today, and I don't think it's reasonable to start requiring it now.